python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_withoutobservedinputs.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+Calibration without observed inputs
+===================================
+"""
+# %%
+# The goal of this example is to present the calibration of a parametric model which 
+# does not have observed inputs.
+# We are going to see how to use the :meth:`~openturns.Sample` class 
```

Comment:
```suggestion
# We are going to see how to use the :class:`~openturns.Sample` class 
```
I am surprised that "meth" works as a keyword here, but it should not be used in this context because it creates inappropriate parentheses.